### PR TITLE
Added script to disable all invites on a conductor

### PR DESCRIPTION
**Pull request by j-williams-tn**

### CODE REVIEW

File: disable_invites/disable_invites.rb [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+  end
+end
+
+# returns array of all invite recipient ids
+def gather_recipient_ids
+  url = "/api/v1/invite_recipients"
+  response = @req.http_get(url)
+  result = []
+  if ['200'].include?(response.code)
+    body = JSON.parse(response.body)
```
Comment (bryanskene) [inline disable_invites/disable_invites.rb:63]: Can response.body ever be nil? The JSON parse would blow up. If not then no worries.

File: disable_invites/disable_invites.rb [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+
+require 'optparse'
+require 'net/http'
+require 'json'
+require 'fileutils'
+require 'time'
+require 'openssl'
+
+class String
+  # colorization
```
Comment (bryanskene) [inline disable_invites/disable_invites.rb:9]: ooh this is cool, I'm stealing it.

File: disable_invites/disable_invites.rb [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+config_params = JSON.parse(File.read(config)) rescue nil
+if config_params.nil?
+  puts("ERROR: failed to parse config in #{config}".red)
+  exit(1)
+end
+
+# validate API config params
+conductor = config_params['conductor_url']
+if conductor.nil? || !conductor.is_a?(String)
+  puts("ERROR: config param conductor_url must be a valid string".red)
```
Comment (bryanskene) [inline disable_invites/disable_invites.rb:121]: Maybe make a method for this repetitive code and even extend the options parsing to deal with it for you. Not important for this script though.

**Review by bryanskene — APPROVED** [review]
Looks great Jared. Thanks for busting it out.

File: disable_invites/disable_invites.rb (hunk 3 above)
Comment (j-williams-tn) [reply disable_invites/disable_invites.rb:121]: Good idea!

File: disable_invites/disable_invites.rb (hunk 1 above)
Comment (j-williams-tn) [reply disable_invites/disable_invites.rb:63]: This should never be nil. If you get a 200 it will either be an empty array or an array of objects.

**Pull request merged**